test/parallel/test-url-invalid-file-url-path-input.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+// This tests that url.fileURLToPath() throws ERR_INVALID_FILE_URL_PATH
+// for invalid file URL paths along with the input property.
+
+const { isWindows } = require('../common');
+const assert = require('assert');
+const url = require('url');
+
+const inputs = [];
+
+if (isWindows) {
+  inputs.push('file:///C:/a%2F/', 'file:///C:/a%5C/', 'file:///?:/');
+} else {
+  inputs.push('file:///a%2F/');
+}
+
+for (const input of inputs) {
```

Comment:
Nit:

Why can't we keep this in the existing test file (test/parallel/test-url-fileurltopath.js). Adding a new test file will only add 50-150ms delay to running the test (because of spawn_process) with almost no benefit. Am I missing something for the purpose of this?